### DISCUSSION TITLE
Manage nicegui apps with uwsgi emperor

### DISCUSSION THREAD

**Discussion by Aluriak** [zauberzeug/nicegui#1979] [Q&A] [answered]
### Question

Hi !

My team is developping multiple web interfaces, and i need them to be automatically deployed.

For the moment, we use uwsgi, allowing us to deploy django, flask and ror apps. (uwsgi works behind nginx)

I didn't found any doc for how to interface nicegui with uwsgi (or any other CGI).

Can you help me find a way to automate nicegui deployment ?

Best regards,

**Comment by falkoschindler** [comment, marked as answer]
Sorry, I don't know uwsgi emperor.
But NiceGUI is built on FastAPI. So if you find a way to deploy a FastAPI app, NiceGUI should be very similar.
